src/frontend/src/lib/components/send/QRCodeModal.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+	import { i18n } from '$lib/stores/i18n.store';
+	import {
+		QRCodeReaderModal,
+		WizardModal,
+		type WizardStep,
+		type WizardSteps
+	} from '@dfinity/gix-components';
+	import { type Token } from '@dfinity/utils';
+	import { toastsError } from '$lib/stores/toasts.store';
+	import type { QrResponse, QrStatus } from '$lib/types/qr-code';
+	import { decodeQrCode } from '$lib/utils/qr-code.utils';
+
+	export let steps: WizardSteps;
+	export let currentStep: WizardStep | undefined = undefined;
+
+	const STEP_QRCODE = 'QRCode';
```

Comment:
I was re-thinking about this:

I am not sure about being a property, because it should be just an additional feature for each of the modals (IC and ETH, and future ones). So i would just use it internally in the QRCodeModal component.

The parent component would not need it, right? The only way to trigger this modal step would be through `scanQrCode` function.

I will create an enum for that (so that it is not an hardcoded string), but I think not to make it a property.

What do you think?

Reply:
Nevermind my comment actually I think I mixed Wizard steps and Progression steps. We can iterate after the PR if required anyway. Sorry for the noise.